rules/gobgp.mk

```diff
@@ -1,5 +0,0 @@
-# gobgp package
```

Comment:
>gobgp [](http://example.com/codeflow?start=2&length=5)

gobgp itself is not related to quagga. Why remove?

Reply:
Undid removal for gobgp.

Reply:
let's keep gobgp and p4. only remove quagga.